var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -89,6 +90,17 @@ def url_for_version(self, version):
         else:
             return self._urlfmt.format(version.up_to(2), version, '')
 
+    def setup_environment(self, spack_env, run_env):
+        if os.path.isdir(self.prefix.lib64):
+            lib_dir = self.prefix.lib64
+        else:
+            lib_dir = self.prefix.lib
+        paraview_version = 'paraview-%s' % self.spec.version.up_to(2)
+        run_env.prepend_path('LIBRARY_PATH', join_path(lib_dir,
```

Comment:
I think it's worth mentioning in a comment that the fix suggested by @davydden 

> what if you add -DCMAKE_INSTALL_LIBDIR=self.prefix.lib? That should works CMake to install into prefix/lib.

from https://github.com/LLNL/spack/issues/5735#issuecomment-336249438 didn't work, and that there isn't a way to tell Spack to RPATH these special lib directories.